libs/SmartSync/src/com/salesforce/androidsdk/smartsync/manager/SyncManager.java

```diff
@@ -289,6 +291,20 @@ private void updateSync(SyncState sync, SyncState.Status status, int progress, S
     		sync.setStatus(status);
     		if (progress != UNCHANGED) sync.setProgress(progress);
     		sync.save(smartStore);
+
+            switch (status) {
+
+                case NEW:
+                    break;
+                case RUNNING:
+                    runningSyncIds.add(sync.getId());
+                    break;
+                case DONE:
```

Comment:
Meant to ask this earlier, but doesn't it have to be removed from the map when it gets to the ```DONE``` status?